### PR TITLE
Rebased hashlock

### PR DESCRIPTION
**Pull request by benjaminbollen**
Tests need to be corrected

### CODE REVIEW

File: test/Protocol.js [hunk 1]
```diff
@@ -251,7 +251,7 @@ contract('OpenST', function(accounts) {
 		    it("register utility token on value chain", async() => {
 
 		     	const result = await registrarVC.registerUtilityToken(openSTValue.address, symbol, name, conversionRate, conversionRateDecimals,
-		    		CHAINID_UTILITY, requester, registeredBrandedTokenUuid, { from: intercommVC });
+		    		CHAINID_UTILITY, stakerVC, registeredBrandedTokenUuid, { from: intercommVC });
```
Comment (benjaminbollen) [inline test/Protocol.js]: this is not necessary I think, but does not harm

File: test/OpenSTUtility.js [hunk 2]
```diff
@@ -710,13 +710,7 @@ contract('OpenSTUtility', function(accounts) {
 					await OpenSTUtility_utils.checkRevertedMintEvent(result.logs[0], checkBtUuid, stakingIntentHash, accounts[0],
 						accounts[0], AMOUNT_BT);
 				})
-			})
-
-			it('successfully revert Minting', async () => {
```
Comment (benjaminbollen) [inline test/OpenSTUtility.js]: any idea why this test is failing?

File: test/Protocol.js [hunk 3]
```diff
@@ -372,7 +372,7 @@ contract('OpenST', function(accounts) {
 			it("transfer STPrime to Redeemer", async() => {
 
 				var redeemerBalanceBeforeTransfer = await web3.eth.getBalance(redeemer).toNumber();
-				result = await web3.eth.sendTransaction({ from: staker, to: redeemer, value: FUND_AMOUNT_STPRIME ,gasPrice: '0x12A05F200' });
+				result = await web3.eth.sendTransaction({ from: stakerVC, to: redeemer, value: FUND_AMOUNT_STPRIME ,gasPrice: '0x12A05F200' });
```
Comment (benjaminbollen) [inline test/Protocol.js]: good catch

File: test/OpenSTUtility.js (hunk 2 above)
Comment (benjaminbollen) [reply test/OpenSTUtility.js]: ah it was duplicated 🤦‍♂️

**Comment by benjaminbollen** [comment]
LGTM! thanks, fresh eyes helped

File: test/OpenSTUtility.js (hunk 2 above)
Comment (abhayks1) [reply test/OpenSTUtility.js]: yup
also there were syntax errors with curly brackets while rebasing.

File: test/Protocol.js (hunk 1 above)
Comment (abhayks1) [reply test/Protocol.js]: +1

File: test/Protocol.js (hunk 3 above)
Comment (abhayks1) [reply test/Protocol.js]: +1

**Pull request merged**